worlds/dark_souls_3/Options.py

```diff
@@ -26,93 +101,124 @@ class NoSpellRequirementsOption(Toggle):
-    range_start = 1
+    display_name = "Maximum Level of +10 Weapons"
+    range_start = 0
     range_end = 10
     default = 10
 
 
 class LateBasinOfVowsOption(Toggle):
-    """Force the Basin of Vows to be located as a reward of defeating Pontiff Sulyvahn. It permits to ease the
-    progression by preventing having to kill the Dancer of the Boreal Valley as the first boss"""
+    """This option makes it so the Basin of Vows is still randomized, but guarantees you that you wont have to venture into
+    Lothric Castle to find your Small Lothric Banner to get out of High Wall of Lothric. So you may find Basin of Vows early, 
+    but you wont have to fight Dancer to find your Small Lothric Banner."""
     display_name = "Late Basin of Vows"
 
 
-class EnableProgressiveLocationsOption(Toggle):
-    """Randomize upgrade materials such as the titanite shards, the estus shards and the consumables"""
-    display_name = "Randomize materials, Estus shards and consumables (+196 checks/items)"
+class LateDLCOption(Toggle):
+    """This option makes it so you are guaranteed to find your Small Doll without having to venture off into the DLC, 
+    effectively putting anything in the DLC in logic after finding both Contraption Key and Small Doll, 
+    and being able to get into Irithyll of the Boreal Valley"""
+    display_name = "Late DLC"
 
 
 class EnableDLCOption(Toggle):
     """To use this option, you must own both the ASHES OF ARIANDEL and the RINGED CITY DLC"""
-    display_name = "Add the DLC Items and Locations to the pool (+81 checks/items)"
-
-
-dark_souls_options: typing.Dict[str, type(Option)] = {
+    display_name = "Enable DLC"
+
+
+dark_souls_options: typing.Dict[str, Option] = {
+    "enable_weapon_locations": RandomizeWeaponLocations,
+    "enable_shield_locations": RandomizeShieldLocations,
+    "enable_armor_locations": RandomizeArmorLocations,
+    "enable_ring_locations": RandomizeRingLocations,
+    "enable_spell_locations": RandomizeSpellLocations,
+    "enable_key_locations": RandomizeKeyLocations,
+    "enable_boss_locations": RandomizeBossSoulLocations,
+    "enable_npc_locations": RandomizeNPCLocations,
+    "enable_misc_locations": RandomizeMiscLocations,
+    "enable_health_upgrade_locations": RandomizeHealthLocations,
+    "enable_progressive_locations": RandomizeProgressiveLocationsOption,
+    "pool_type": PoolTypeOption,
+    "guaranteed_items": GuaranteedItemsOption,
     "auto_equip": AutoEquipOption,
     "lock_equip": LockEquipOption,
     "no_weapon_requirements": NoWeaponRequirementsOption,
-    "randomize_weapons_level": RandomizeWeaponsLevelOption,
-    "randomize_weapons_percentage": RandomizeWeaponsLevelPercentageOption,
+    "randomize_infusion": RandomizeInfusionOption,
+    "randomize_infusion_percentage": RandomizeInfusionPercentageOption,
+    "randomize_weapon_level": RandomizeWeaponLevelOption,
+    "randomize_weapon_level_percentage": RandomizeWeaponLevelPercentageOption,
```

Comment:
This will break existing yamls. Is that fine by everyone?

Reply:
My opinion is that the majority of people will be remaking their yamls anyway, so it seems like the right opportunity to make these setting names more consistent and intuitive from a native English speaker's perspective.